src/corehost/error_codes.h

```diff
@@ -18,7 +19,6 @@ enum StatusCode
     CoreHostLibMissingFailure           = 0x80008083,
     CoreHostEntryPointFailure           = 0x80008084,
     CoreHostCurHostFindFailure          = 0x80008085,
-    CoreHostResolveModeFailure          = 0x80008086,
```

Comment:
Thoughts on having a comment for unused values like `// unused = 0x80008086`? (we could fill them in as we add more status codes instead of having the gaps?)